bigml/ensemble.py

```diff
@@ -253,12 +280,63 @@ def list_models(self):
         """
         return self.model_ids
 
+    def predict_probability(self, input_data, method=PROBABILITY_CODE,
```

Comment:
I see that you use the .predict method inside of this one. What about the rest of arguments in the .predict method? Most of them deal with the information included in the output and its format, but in particular the "by_name" argument is telling whether the input uses IDs or field names as keys. This one should be maintained, if I'm not mistaken.